custom_components/waste_collection_schedule/waste_collection_schedule/source/blackpool_gov_uk.py

```diff
@@ -0,0 +1,75 @@
+import requests
+
+from datetime import datetime
+from waste_collection_schedule import Collection
+
+TITLE = "Blackpool Council"
+DESCRIPTION = "Source for blackpool.gov.uk"
+URL = "https://blackpool.gov.uk"
+
+BINDAY_URL_TOKEN = "https://api.blackpool.gov.uk//api/bartec/security/token"
+BINDAY_URL_SCHEDULE = "https://api.blackpool.gov.uk//api/bartec/collection/PremiseJobs"
+ICON_MAP = {
+    "Blue Bin": "mdi:recycle",
+    "Grey bin or Red sack": "mdi:trash-can",
+    "Green bin or Garden waste": "mdi:leaf",
+    "Commercial Refuse": "mdi:trash-can",
+}
+
+
+class Source:
+    """Waste collection source for blackpool.gov.uk"""
+    def __init__(self, postcode=None, address=None, uprn=None):
```

Comment:
Please remove postcode and address - they are not described in the docu and therefore obviously obsolete.